internal/server/v2/observation/contained_in.go

```diff
@@ -210,6 +215,10 @@ func FetchContainedIn(
 					obsByEntity := result.ByVariable[variable].ByEntity
 					data, ok := btData[variable]
 					if !ok || data == nil {
+						// If data is missing, attach empty child places.
+						for _, place := range childPlaces {
+							obsByEntity[place] = &pbv2.EntityObservation{}
+						}
```

Comment:
feels like this should be part of Flume logic. Right now Flume does not return place with no data. This is a behavior discrepancy. 

I actually think it makes sense to not have an entry for a child place if there is no data in all scenarios.

Reply:
Should I add this to the flume instead?

Reply:
I still don't think we should add places with no data. Imagine fetching cities in USA for a custom DC variable which has data for 10 places. That would have thousands of empty entries.

I think we should gvc and understand why the child places are needed..